packages/server/src/server.ts

```diff
@@ -208,14 +133,7 @@ export class ProsopoServer {
             // We don't care about recency with PoW challenges as they are single use, so just return the verified result
             return result.verified
         }
-        const recent = await verifyRecency((await this.getContractApi()).api, blockNumber, timeouts.image.cachedTimeout)
-
-        if (!recent) {
```

Comment:
do we still have this check?

Reply:
Yeah it should still do something like this before it calls out to the provider

Reply:
added a timeout check